tvshows/src/components/shared/SidebarNavigation/SidebarNavigation.tsx

```diff
@@ -48,7 +48,17 @@ export const SidebarNavigation = () => {
         textAlign="right"
         pr={5}
       >
-        <Button bg="#110429" textColor="aliceblue" mr={3} onClick={() => {}}>
+        <Button
+          bg="#110429"
+          textColor="aliceblue"
+          mr={3}
+          onClick={() => {
+            localStorage.removeItem("client");
+            localStorage.removeItem("access-token");
+            localStorage.removeItem("uid");
+            window.location.href = "/login"; 
```

Comment:
This is OK, but think about how we can utilise the AuthRedirect component for this redirect.

Reply:
onClick={() => {
            localStorage.removeItem("client");
            localStorage.removeItem("access-token");
            localStorage.removeItem("uid");
            <AuthRedirect to="/login" condition={"loggedIn"} />;
          }}
          
 when I try with AuthRedirect, I get error message displayed and only after refreshing the page I get redirected to Sign In form,